tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -68,18 +68,18 @@ If re-drawing is required because a state or something else changed,
 simply call :ref:`CanvasItem.update() <class_CanvasItem_update>`
 in that same node and a new _draw() call will happen.
 
-Here is a little more complex example. A texture variable that will be
+Here is a little more complex example - a texture variable that will be
```

Comment:
Perhaps an em dash character will be better here? (—)

Reply:
@bojidar-bg Typographically, you are correct (assuming the code sample is interpreted as a clause following the colon). However, I'm afraid the character will not display properly for some readers, as the em dash is not part of the ASCII character set. (Aside: typewriters have disgraced the conventions established by the printing press.) If Unicode is known to be fully supported, then an em dash could be substituted there.

Reply:
Correct me if I'm wrong, but I believe we use the em dash elsewhere in the docs as well. Additionally, Unicode tends to display on most devices that I've seen, with the usual exceptions being *really* old phones (y'know, the ones that were made before Android and iOS). I don't believe there's much to be concerned about when it comes to Unicode support.

Reply:
I just suggested it as some people don't know that em and en dash symbols exist in Unicode, and thus reuse the hyphen for them. Anyway, I rarely use dashes myself, so I like the comma more. :)